webrtc-stats.html

```diff
@@ -2456,6 +2477,55 @@ <h2>
           </section>
         </div>
       </section>
+
+      <section id="playoutstats-dict*">
+        <h3>
+          {{RTCAudioPlayoutStats}} dictionary
+        </h3>
+        <p>
+          Only applicable if the playout path represents an audio devcice.
+          Represents one playout path, which may or may not be after mixing.
```

Comment:
...may not be after mixing the inbound audio streams

(clarifying that this may represent the mixed audio stream and there is no way to know which incoming stream may have caused the issue)